packages/ui/src/SelectField.module.scss

```diff
@@ -95,6 +95,19 @@ $select: ':global .hz-select-field__control';
   }
 }
 
+// Styles for MultiValue chips - hover and focused states.
+// We inject `.multiValueIsFocused` from js (it does not have a :focused state as its tabIndex is not given).
+.multiValueIsFocused {
+  :global .hz-select-field__multi-value__remove {
+    background-color: c.$colorWarning;
+    border: 1px solid c.$colorAccessibilityOutline;
+  }
+}
+:global .hz-select-field__multi-value__remove:hover {
+  background-color: c.$colorWarning;
+  border: 1px solid c.$colorPrimary;
```

Comment:
```suggestion
  border: c.$borderWidth solid c.$colorPrimary;
```

Reply:
Should we also add a `1px` transparent border for the unhovered state, so the cell does not jump when hovered?